ch01.adoc

```diff
@@ -228,3 +228,14 @@ Briefly the new attributes allow:
 * Description of climatological statistics.
 * Data compression for variables with missing values.
 
+
+[[ugrid-conventions, Section 1.6, "UGRID Conventions"]]
+=== UGRID Conventions
+
+These conventions implicitly incorporate parts of the UGRID conventions for storing unstructured (or flexible mesh) data in netCDF files using mesh topologies <<UGRID>>.
+Only version 1.0 of the UGRID conventions is allowed.
```

Comment:
Ah, it looks like my question here might be answered in https://github.com/cf-convention/cf-conventions/issues/153#issuecomment-702679434. I still think it would be good to clarify that somewhere in the text.

Reply:
Resolved in 5a8fc6f (with brief discussion over on https://github.com/cf-convention/cf-conventions/issues/153#issuecomment-1759132807). Thanks.